README.md

```diff
@@ -80,6 +80,9 @@ string; default: null; Cypress specs root relative to package json. [More detail
 #### `options.outputTarget`
 object; default: null; Output logs to files. [More details](#logging-to-files).
 
+#### `options.outputVerbose`
+boolean; default: true; Indicates whether 'Wrote custom logs to ..' appears on the console or not.
```

Comment:
Let's consider this something more general. Just write as description: `Toggles verbose output.` Later on we might use if for other similar output.